lib/features/artifacts.js

```diff
@@ -71,8 +71,19 @@ export default class Artifacts {
 
       return;
     }
+
+    if (taskHandler.isAborted()) return;
```

Comment:
you might want to try `contentStream.destroy()` or something... not sure the right thing is...